src/pages/signin/SignInPageLayout/index.js

```diff
@@ -10,9 +10,9 @@ const propTypes = {
 const SignInPageLayout = props => (
     !props.isSmallScreenWidth
         // eslint-disable-next-line react/jsx-props-no-spreading
```

Comment:
Doesn't look like there's any props spreading happening here anymore, can you try getting rid of these `eslint-disable-next-line` statements?

Reply:
You're right. No spreading. Removed and no lint error. I am not sure why it was added earlier.